internal/handlers/boot_artifacts.go

```diff
@@ -37,6 +37,12 @@ func parseArtifact(path, arch string) (string, error) {
 		} else {
 			artifact = "vmlinuz"
 		}
+	case "ins-file":
```

Comment:
Can this just be `ins`?
What does INS stand for? Does it make sense to be used alone or does saying "INS file" make more sense?

For example, I wouldn't say "kernel file" or "rootfs file" I would just say "the kernel" or "the rootfs".